db/migrate/20190424153057_create_restaurants.rb

```diff
@@ -0,0 +1,24 @@
+class CreateRestaurants < ActiveRecord::Migration[6.0]
+  def up
+    enable_extension 'uuid-ossp'
```

Comment:
1. Please move this to another migrations, and run it under `def change` instead of `def up`. This is good practice to enable extensions on different migrations.
1. where is `pgcrypto`?
1. If you aren't using `uuid_generate_v4`, you don't need _uuid-ossp_.